src/test/java/data/TaskManagerTest.java

```diff
@@ -0,0 +1,96 @@
+package data;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import java.time.LocalDate;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static data.TaskManager.addTask;
+import static data.TaskManager.updateTask;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+class TaskManagerTest {
+    private TaskManager taskManager;
+
+    @BeforeEach
+    void setUp() {
+        taskManager = new TaskManager();
+    }
```

Comment:
Add this:

@AfterEach
void resetTaskManager() {
    taskManager = null;
}

hopefully resolves the CI failure